app/models/BakeryTransactor.scala

```diff
@@ -0,0 +1,13 @@
+package models
+
+import cats.effect.IO
+import doobie.Transactor
+
+class BakeryTransactor {
+  val xa: Transactor[IO] = Transactor.fromDriverManager[IO](
+    "org.postgresql.Driver", // postgres driver
+    "jdbc:postgresql://localhost:5432/pass_bakery_db", // connect URL
+    "user",
+    "pass"
+  )
+}
```

Comment:
I guess this works. I was hoping there was a way to have a transactor instance just injected into the bakerydb, but if this shell class makes the transactor available then I'm okay with that. 

Stretch goal: notice how you are hardcoding the configuration of your transactor, specifically the database one. Can you instead read those values from environment variables?

Reply:
Changed this to inject the play configuration so the values are read from application.conf instead of being hard coded in.